src/gl/interface/gl-objects-buffers.ads

```diff
@@ -57,17 +60,25 @@ package GL.Objects.Buffers is
    procedure Draw_Elements (Mode : Connection_Mode; Count : Types.Size;
                             Index_Type : Unsigned_Numeric_Type;
                             Element_Offset : Natural := 0);
-
+   procedure Draw_Elements_Instanced (Mode : Connection_Mode; Count : UInt;
+                                      Index_Type : Unsigned_Numeric_Type;
+                                      Element_Offset : UInt := 0;
+                                      Instance_Count : UInt := 0);
    procedure Invalidate_Data (Object : in out Buffer);
    procedure Invalidate_Sub_Data (Object : in out Buffer;
                                   Offset, Length : Long_Size);
 
+   subtype Texture_Buffer_Target is Buffer_Target;
```

Comment:
Please make this a derived type, not a subtype. Only a new type will disallow calling `Allocate` below with a `Buffer_Target`.

Reply:
According to my reference manual this should work:
~~~
 type Texture_Buffer_Target is limited new Buffer_Target with null record;
~~~
but it produces an error message:
~~~
completion of nonlimited type  cannot be limited
~~~
What am I missing?

Reply:
Fixed

Reply:
The PR still contains the subtype.